js/components/welcome.js

```diff
@@ -4,7 +4,7 @@ define([
     'appConfig',
     'services/AuthAPI',
 	'utils/BemHelper',
-    'less!welcome.less'
+    'less!components/welcome.less'
```

Comment:
Same comment here: welcome.less belongs to welcome module, it should use a relative path.

Reply:
Ah, I think the issue with the less!welcome.less is that it should ahev been less!./welcome.less. Could you try that?